### ISSUE TITLE
Depends on ansiblebit.launchpad-ppa-webupd8 even on Centos

### ISSUE THREAD

**Issue by marsbard** [ansiblebit/oracle-java#30]
Installing under CentOs gives the following error:
```
ERROR! the role 'ansiblebit.launchpad-ppa-webupd8' was not found in /home/klo/build-resources/ansible/roles:/etc/ansible/roles:/home/ko/build-resources/ansible/roles:/home/klo/build-resources/ansible

The error appears to have been in '/home/klo/build-resources/ansible/roles/ansiblebit.oracle-java/meta/main.yml': line 42, column 5, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

dependencies:
  - role: ansiblebit.launchpad-ppa-webupd8
    ^ here
```

**Comment by marsbard** [comment]
```
$ ansible all -i targethost, -m setup | grep ansible_distribution
        "ansible_distribution": "CentOS", 
        "ansible_distribution_major_version": "7", 
        "ansible_distribution_release": "Core", 
        "ansible_distribution_version": "7.2.1511",
```

**Comment by steenzout** [comment]
@marsbard which ansible version did you use?

on the `meta/main.yml` I have defined this

```
dependencies:
  - role: ansiblebit.launchpad-ppa-webupd8
    when: ansible_distribution | lower == 'debian' or ansible_distribution | lower == 'ubuntu'
```

and it was working in the past but that may have changed.

if you tell me the version you use I can simulate the problem on my laptop.

**Comment by steenzout** [comment]
@marsbard any input? please see my previous reply.

**Comment by marsbard** [comment]
I'll see if I get a chance to look at it tomorrow when I'm at work (although it's slipped on my priority list right now...)

**Comment by steenzout** [comment]
_no answer back_.

please reopen whenever you get some extra info.

cheers!

**Comment by steenzout** [comment]
the role dependency was removed so the issue is definitely resolved from version >= 5.0.0.
